internal/command/console/console.go

```diff
@@ -56,6 +59,81 @@ func New() *cobra.Command {
 			Description: "Unix username to connect as",
 			Default:     ssh.DefaultSshUsername,
 		},
+		flag.String{
+			Name:        "image",
+			Shorthand:   "i",
+			Description: "image to use (default: current release)",
+		},
+		flag.StringArray{
+			Name:        "env",
+			Shorthand:   "e",
+			Description: "Set of environment variables in the form of NAME=VALUE pairs. Can be specified multiple times.",
+		},
+		flag.String{
+			Name:        "entrypoint",
+			Description: "ENTRYPOINT replacement",
+		},
+		flag.Bool{
+			Name:        "build-only",
+			Description: "Only build the image without running the machine",
+			Hidden:      true,
+		},
+		flag.Bool{
+			Name:        "build-remote-only",
+			Description: "Perform builds remotely without using the local docker daemon",
+			Hidden:      true,
+		},
+		flag.Bool{
+			Name:        "build-local-only",
+			Description: "Only perform builds locally using the local docker daemon",
+			Hidden:      true,
+		},
+		flag.Bool{
+			Name:        "build-nixpacks",
+			Description: "Build your image with nixpacks",
+			Hidden:      true,
+		},
+		flag.String{
+			Name:        "dockerfile",
+			Description: "Path to a Dockerfile. Defaults to the Dockerfile in the working directory.",
+		},
+		flag.StringArray{
+			Name:        "build-arg",
+			Description: "Set of build time variables in the form of NAME=VALUE pairs. Can be specified multiple times.",
+			Hidden:      true,
+		},
+		flag.String{
+			Name:        "image-label",
+			Description: "Image label to use when tagging and pushing to the fly registry. Defaults to \"deployment-{timestamp}\".",
+			Hidden:      true,
+		},
+		flag.String{
+			Name:        "build-target",
+			Description: "Set the target build stage to build if the Dockerfile has more than one stage",
+			Hidden:      true,
+		},
+		flag.Bool{
+			Name:        "no-build-cache",
+			Description: "Do not use the cache when building the image",
+			Hidden:      true,
+		},
+		flag.String{
+			Name:        "command",
+			Shorthand:   "C",
+			Default:     "",
+			Description: "command to run on SSH session",
+		},
+		flag.StringSlice{
+			Name:      "port",
+			Shorthand: "p",
+			Description: `Publish ports, format: port[:machinePort][/protocol[:handler[:handler...]]])
+		i.e.: --port 80/tcp --port 443:80/tcp:http:tls --port 5432/tcp:pg_tls
+		To remove a port mapping use '-' as handler, i.e.: --port 80/tcp:-`,
+		},
+		flag.Bool{
+			Name:        "skip-dns-registration",
+			Description: "Do not register the machine's 6PN IP with the internal DNS system",
+		},
```

Comment:
Console machines already have DNS registration skipped by default. Maybe add `Default: true` so that it gets documented in the help text?

Reply:
Changing it to default true is a good idea; but I'm going to also mark it Hidden at the same time.  That way if there is a need it can be shared with others but more likely the topic won't come up.

Good catch!